app/src/main/java/objectorientation/Box.java

```diff
@@ -0,0 +1,8 @@
+package objectorientation;
+
+public class Box {
+    int length,width,height;
+    public int volume(){
+        return length*width*height;
```

Comment:
we can also add the colour of the box 

public int colour() { int colour= blue; return colour; }

Reply:
ok